tests/neural_nets_factory.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+
+from sbi.utils.get_nn_models import classifier_nn, likelihood_nn, posterior_nn
+
+
+@pytest.mark.parametrize(
+    "model", ["linear", "mlp", "resnet"], ids=["linear", "mlp", "resnet"]
```

Comment:
I have not seen the `ids` arg before, what is it doing?

Reply:
So `ids` gives human-readable names that can be used to further subselect tests. For example `pytest tests/neural_nets_factory.py::test_deprecated_import_classifier_nn` runs all 3, but `pytest tests/neural_nets_factory.py::test_deprecated_import_classifier_nn -k "linear"` only the one with the linear model. The real power arises when you give ids to every parameter and want to check on the scenario.
I enjoy the integration with my IDE since it makes it faster to see which test combinations crash.